src/main/java/core/basesyntax/AreaCalculator.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface AreaCalculator {
+    double areaCalculator();

```

Comment:
```suggestion
    double calculateArea();
```

Methods should answer the question "what to do", i.e. must be a verb